CONTRIBUTING.rst

```diff
@@ -270,90 +234,21 @@ Running System Tests
 - For datastore query tests, you'll need stored data in your dataset.
   To populate this data, run::
 
-   $ python system_tests/populate_datastore.py
+   $ python datastore/tests/system/utils/populate_datastore.py
 
 - If you make a mistake during development (i.e. a failing test that
   prevents clean-up) you can clear all system test data from your
   datastore instance via::
 
-   $ python system_tests/clear_datastore.py
-
-System Test Emulators
```

Comment:
@lukesneeringer Why did you delete this? Running in the emulators is very useful.